app/components/JobList.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { Component, PropTypes } from 'react';
```

Comment:
Put this code directly in the `DisplayedJobs` component. Just because it's a component that interacts with Redux doesn't mean it should be separate from `render` code.